tests/Behat/Context/Shop/CustomerContext.php

```diff
@@ -32,18 +35,33 @@ final class CustomerContext implements Context
 
     private IriConverterInterface $iriConverter;
 
+    private EntityManagerInterface $entityManager;
+
     public function __construct(
         GraphqlClientInterface $client,
         SharedStorageInterface $sharedStorage,
         UserRepositoryInterface $userRepository,
         ShopUserTokenFactoryInterface $tokenFactory,
         IriConverterInterface $iriConverter,
+        EntityManagerInterface $entityManager,
     ) {
         $this->client = $client;
         $this->sharedStorage = $sharedStorage;
         $this->userRepository = $userRepository;
         $this->tokenFactory = $tokenFactory;
         $this->iriConverter = $iriConverter;
+        $this->entityManager = $entityManager;
+    }
+
+    /**
+     * @BeforeScenario
+     */
+    public function purgeDatabase()
```

Comment:
Is this needed? If so, could you tell me, why?

Reply:
i had problems with unique emails while testing.